### PR TITLE
Adding the install/build steps for Fedora

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: docs/install/Fedora.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#INSTALL
+atomic can be installed through below methods
+
+##YUM/DNF
+atomic RPM is available on Fedora 21 and later. So a simple `yum/dnf install atomic` will install it fine.
+
+##Make
+On Fedora 21, You need to install required build dependencies 
+```
+yum install pylint python-docker-py go-md2man
```
Comment (cgwalters) [inline docs/install/Fedora.md]: Why not recommend `yum-builddep atomic`?  That's what I do to get a starting point.

Comment (lsm5) [inline docs/install/Fedora.md]: ah yup, `yum-builddep atomic` or `dnf builddep atomic` would work better than yum install.

I think all yum commands including `yum-builddep` have been deprecated starting with f22, so might need to replace with `dnf builddep`. yum would still work but will just throw a deprecation warning

**Comment by LalatenduMohanty** [comment]
@cgwalters @lsm5 Thanks for the suggestion. I will update the PR. However `dnf builddep atomic` is as of now not working as expected (refer https://github.com/rpm-software-management/dnf-plugins-core/blob/master/doc/builddep.rst)

# dnf builddep atomic
Using metadata from Mon Apr 27 12:21:22 2015
Error: Failed to open: 'atomic', not a valid spec file.

**Comment by lsm5** [comment]
ahh, that's probably only a rawhide thing in that case, my bad :|

**Comment by rhatdan** [comment]
@lsm5 Is this fixed.  @LalatenduMohanty This should probably be a bugzilla not in here.

**Comment by lsm5** [comment]
@rhatdan this is just a documentation update to be added to the upstream repo itself and not a bug.

**Comment by rhatdan** [comment]
ok

**Comment by lsm5** [comment]
@rhatdan could you please re-open (and hopefully merge) this?

**Pull request merged**